actors/abi/bigint/bigint.go

```diff
@@ -0,0 +1,250 @@
+package big
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/big"
+
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/polydawn/refmt/obj/atlas"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+)
+
+const BigIntMaxSerializedLen = 128 // is this big enough? or too big?
+
+func init() {
+	cbor.RegisterCborType(atlas.BuildEntry(Int{}).Transform().
+		TransformMarshal(atlas.MakeMarshalTransformFunc(
+			func(i Int) ([]byte, error) {
+				return i.cborBytes(), nil
+			})).
+		TransformUnmarshal(atlas.MakeUnmarshalTransformFunc(
+			func(x []byte) (Int, error) {
+				return fromCborBytes(x)
+			})).
+		Complete())
+}
+
+type Int struct {
+	*big.Int
+}
+
+func NewInt(i int64) Int {
+	return Int{big.NewInt(0).SetInt64(i)}
+}
+
+func BigFromBytes(b []byte) Int {
```

Comment:
These methods all stutter given the package name, and could be more informative, e.g. `big.IntFromBytes`, `big.Mul`.